### PR TITLE
Pie chart

### PR DESCRIPTION
**Pull request by judyzaratan**
* Adds in charging pie chart to markers
* Clubhouse ticket [ch3287]

### CODE REVIEW

**Comment by judyzaratan** [comment]
Charging pies rendering on Dash -- I have another update where I have added additional properties for the charging pie to show...It will not show when you run Dash locally with this `shared-components` repository.


screen shot 2018-10-22 at 12 09 23 pm

File: src/marker/SiteMarker/index.js [hunk 1]
```diff
@@ -1,47 +1,163 @@
...
-      return COLOR.LIGHT_BLUE
+      return COLOR.BLUE_LIGHT
     }
-    return COLOR.BLUE
+    return COLOR.BLUE_MEDIUM
   }
 
   render() {
-    const { text, onClick, issues } = this.props
+    const { level, pie, pie1, pie2, text, onClick, issues } = this.props
```
Comment (judyzaratan) [inline src/marker/SiteMarker/index.js]: can we rename pie1 and pie2 to percentL2 and percentL3

Comment (judyzaratan) [reply src/marker/SiteMarker/index.js]: and also look into removing level and pie

File: src/marker/SiteMarker/index.js [hunk 2]
```diff
@@ -12,157 +12,147 @@ class SiteMarker extends Component {
     super(props)
 
     this.state = {
-      height: props.pie !== null ? 50 : 61,
-      width: props.pie !== null ? 39 : 47,
+      height: props.valueL2 !==null && props.valueL3 !== null? 61 : 50,
+      width: props.valueL2 !==null&& props.valueL3 !== null ? 47 : 39,
```
Comment (ste00martin) [inline src/marker/SiteMarker/index.js]: spacing

**Pull request merged**